modules/default/testchart/dynchart.js

```diff
@@ -0,0 +1,175 @@
+/* global Module */
+
+/* Magic Mirror
+ * Module: testchart
+ *
+ * By Chris van Marle
+ * MIT Licensed.
+ */
+
+Module.register("testchart",{
+
+	// Default module config.
+	defaults: {
+	},
+
+	getScripts: function() {
+		return [/*"moment.js",*/ "Chart.js"];
+	},
+
+	start: function() {
+		this.dataid = 0;
+
+		this.chartData = {
+			labels: [],
+			datasets: []
+		}
+
+		var self = this;
+		/*
+		this.refInterval = window.setInterval(function() {
+			var sample = self.config.samples[self.dataid];
+
+			self.chartData.labels.push(sample[0]);
+			for(var i=0; i<sample.length - 1; ++i) {
+				if(i >= self.chartData.datasets.length) {
+					self.chartData.datasets.push([]);
+				}
+				self.chartData.datasets[i].push(sample[i + 1]);
+			}
+			self.updateChartData();
+
+			if(++self.dataid >= self.config.samples.length) {
+				window.clearInterval(self.refInterval);
+			}
+		}, 2000);
+		*/
+	},
+
+	updateChartData: function() {
+		if(this.myChart !== undefined) {
+			this.myChart.data.labels = this.chartData.labels;
+			for(var i=0; i<this.myChart.data.datasets.length && i<this.chartData.datasets.length; ++i) {
+				this.myChart.data.datasets[i].data = this.chartData.datasets[i];
+			}
+			this.myChart.update();
+		}
+	},
+
+	// Override dom generator.
+	getDom: function() {
+		var wrapper = document.createElement("div");
+		this.ctx = document.createElement("canvas");
+		wrapper.appendChild(this.ctx);
+
+		this.myChart = new Chart(this.ctx, this.config.chartjs);
+
+		/*this.myChart = new Chart(this.ctx, {
```

Comment:
The dynchart is still a work in progress. Those comments will of course be removed. 

The sample included can be used as is to show static data, just like helloworld can be used to show a static message.